src/buildMathML.js

```diff
@@ -85,7 +85,7 @@ export const getVariant = function(
     }
 
     const font = options.font;
-    if (!font) {
+    if (!font || font === "mathnormal") {
```

Comment:
I'm not sure what variant `\mathnormal` should use, but as it acts similar to default math font, I used `null`.

Reply:
That should be fine.